src/libraries/System.Private.Xml/src/System/Xml/Core/XmlRawTextWriterGenerator.ttinclude

```diff
@@ -147,66 +147,66 @@ namespace System.Xml
         {
             Debug.Assert(stream != null && settings != null);
 
-            this.stream = stream;
-            this.encoding = settings.Encoding;
+            this._stream = stream;
+            this._encoding = settings.Encoding;
 
             // the buffer is allocated will OVERFLOW in order to reduce checks when writing out constant size markup
             if (settings.Async)
             {
-                bufLen = ASYNCBUFSIZE;
+                _bufLen = ASYNCBUFSIZE;
             }
 
-            <#= BufferName #> = new <#= BufferType #>[bufLen + OVERFLOW];
+            <#= BufferName #> = new <#= BufferType #>[_bufLen + OVERFLOW];
```

Comment:
seems `<#= BufferName #>` need to be updated here https://github.com/dotnet/runtime/blob/b0721f1f8415bc5064f898a1e94747ca8965f009/src/libraries/System.Private.Xml/src/System/Xml/Core/RawTextWriterEncoded.ttinclude#L8

Reply:
might be worth run all template for all generated files and build